Missing Hyperlink in Connection Pooling Config

# Missing Hyperlink in Connection Pooling Config

## On https://app.supabase.io/project/[id]/database/pooling there is a missing hyperlink.

For the Pool Mode Option, you say "To find out the most suitable mode for your use case, click here", but there is no link at "click here".

## To Reproduce
1. Go to https://app.supabase.io/project/[id]/database/pooling


## Expected behavior

There should be a hyperlink to the docs that describes what the Pool Mode options is.

`npm install` for studio currently fails for me on windows with node v14.18.2 and npm v6.14.15.

```
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @sentry/cli@1.71.0 install: `node scripts/install.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @sentry/cli@1.71.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     \npm-cache\_logs\2021-12-29T04_11_21_177Z-debug.log
```

--- a/studio/components/interfaces/Database/Pooling/PgBouncerConfig.tsx
+++ b/studio/components/interfaces/Database/Pooling/PgBouncerConfig.tsx
@@ -107,7 +107,7 @@ const PgbouncerConfig: FC<Props> = ({ projectRef, config }) => {
                   suitable mode for your use case,{' '}
                   <Typography.Link
                     target="_blank"
-                    href="https://supabase.com/blog/2021/04/02/supabase-pgbouncer#pool-modes"
+                    href="https://supabase.com/docs/guides/database/connecting-to-postgres#connection-pool"
                   >
                     click here
                   </Typography.Link>

